sdk/rdbms/azure-mgmt-rdbms/CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Release History
 
+## 9.0.0 (2021-07-01)
+
+**Features**
```

Comment:
@msyyc do you know what it would take to update the changelog automation to start using the changlog guidelines at https://aka.ms/azsdk/guideline/changelogs? It would be good to start to normalize these so things like release notes in https://github.com/Azure/azure-sdk/pull/3106/files#diff-08335e0740b49e683f3701559846980fa57e48206720236dffed325e475ad0d4R376 will match the other PRs. Could you please file and issue to update the generator to update these headings?

Reply:
got it. We would make a study for https://aka.ms/azsdk/guideline/changelogs